redirects-transition.conf

```diff
@@ -348,23 +550,28 @@ rewrite ^/pdf/audit_thresholds_auth_2009-10.pdf https://www.fec.gov/resources/cm
 rewrite ^/pdf/audit_thresholds_unauth_2009-10.pdf https://www.fec.gov/resources/cms-content/documents/2009-2010_audit_materiality_thresholds_title_2_unauthorized.pdf redirect;
 rewrite ^/pdf/audit_threshold_title26_2008.pdf https://www.fec.gov/resources/cms-content/documents/2008_audit_materiality_thresholds_title26_presidential.pdf redirect;
 
+# pdf/ Campaign Guide redirects 
+rewrite ^/pdf/candgui.pdf https://www.fec.gov/resources/cms-content/documents/candgui.pdf redirect;
+rewrite ^/pdf/colagui.pdf https://www.fec.gov/resources/cms-content/documents/colagui.pdf redirect;
+rewrite ^/pdf/partygui.pdf https://www.fec.gov/resources/cms-content/documents/partygui.pdf redirect;
+rewrite ^/pdf/nongui.pdf https://www.fec.gov/resources/cms-content/documents/nongui.pdf redirect;
+
 # pdf/forms/ redirects
 rewrite ^/pdf/forms/fecfrm1ssf.pdf https://www.fec.gov/resources/cms-content/documents/fecfrm1.pdf redirect;
 rewrite ^/pdf/forms/fecfrm1nc.pdf https://www.fec.gov/resources/cms-content/documents/fecfrm1.pdf redirect;
 rewrite ^/pdf/forms/fecfrm1party.pdf https://www.fec.gov/resources/cms-content/documents/fecfrm1.pdf redirect;
 
 # pdf/ general redirects
+rewrite ^/pdf/20year.pdf https://www.fec.gov/resources/cms-content/documents/20yearreport.pdf redirect;
 rewrite ^/pdf/67fr05445.pdf https://www.fec.gov/resources/cms-content/documents/fedreg_notice_2002-01_EO13892.pdf redirect;
 rewrite ^/pdf/Additional_Enforcement_Materials.pdf https://www.fec.gov/resources/cms-content/documents/additional_enforcement_materials.pdf redirect;
 rewrite ^/pdf/commissioners_tips.pdf https://www.fec.gov/resources/cms-content/documents/commissioners_tips_2016_EO13892.pdf redirect;
 rewrite ^/pdf/CoverLetter_20130725.pdf https://www.fec.gov/resources/cms-content/documents/letter_to_Committee_on_House_Administration_July_25_2013.pdf redirect;
 rewrite ^/pdf/eleccoll.pdf https://www.fec.gov/introduction-campaign-finance/understanding-ways-support-federal-candidates/presidential-elections/#electoral-college redirect;
+rewrite ^/pdf/firsttenyearsreport.pdf https://www.fec.gov/resources/about-fec/reports/firsttenyearsreport.pdf redirect;
 
-# pdf/ Campaign Guide redirects 
-rewrite ^/pdf/candgui.pdf https://www.fec.gov/resources/cms-content/documents/candgui.pdf redirect;
-rewrite ^/pdf/colagui.pdf https://www.fec.gov/resources/cms-content/documents/colagui.pdf redirect;
-rewrite ^/pdf/partygui.pdf https://www.fec.gov/resources/cms-content/documents/partygui.pdf redirect;
-rewrite ^/pdf/nongui.pdf https://www.fec.gov/resources/cms-content/documents/nongui.pdf redirect;
+# pdf/nprm/ redirects
+rewrite ^/pdf/nprm/emilyslistrepeal/notice_2009-30.pdf	https://www.fec.gov/resources/legal-resources/rulemakings/nprm/emilyslistrepeal/notice_2009-30.pdf redirect;
```

Comment:
Seems to be multiple spaces here. Should only be one space between urls.

Reply:
Updated